start-docker.sh

```diff
@@ -17,17 +16,17 @@ liquibase.hub.mode=off
 EOF
 
 # copy config and qhana db into volume
-cp -n /app/Config.toml /app/data/
-
+cp -n /app/Config.toml /app/qhana-backend.db /app/data/
```

Comment:
This will probably fail as Config.toml is no longer copied to the docker image.
Also the backend database must be updated in place in the volume (and not created outside then copied into the volume).
This is to ensure that an existing database inside an old volume gets correctly updated by liquibase in an updated docker image.